AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -120,23 +121,33 @@ private var runtimeTargets: [Target] {
         .target(
             name: "AWSSDKCommon",
             dependencies: [.crt],
-            path: "Sources/Core/AWSSDKCommon/Sources"
+            path: "Sources/Core/AWSSDKCommon/Sources/AWSSDKCommon"
         ),
         .target(
             name: "AWSSDKEventStreamsAuth",
             dependencies: [.smithyEventStreamsAPI, .smithyEventStreamsAuthAPI, .smithyEventStreams, .crt, .clientRuntime, "AWSSDKHTTPAuth"],
-            path: "Sources/Core/AWSSDKEventStreamsAuth/Sources"
+            path: "Sources/Core/AWSSDKEventStreamsAuth/Sources/AWSSDKEventStreamsAuth"
         ),
         .target(
             name: "AWSSDKHTTPAuth",
-            dependencies: [.crt, .smithy, .clientRuntime, .smithyHTTPAuth, "AWSSDKChecksums", "AWSSDKIdentity"],
-            path: "Sources/Core/AWSSDKHTTPAuth/Sources"
+            dependencies: [.crt, .smithy, .clientRuntime, .smithyHTTPAuth, .awsSDKIdentityAPI, "AWSSDKChecksums"],
+            path: "Sources/Core/AWSSDKHTTPAuth/Sources/AWSSDKHTTPAuth"
+        ),
+        .target(
+            name: "AWSSDKIdentityAPI",
+            dependencies: [.smithy, .smithyIdentityAPI],
+            path: "Sources/Core/AWSSDKIdentityAPI/Sources/AWSSDKIdentityAPI"
         ),
         .target(
             name: "AWSSDKIdentity",
-            dependencies: [.crt, .smithy, .clientRuntime, .smithyIdentity, .smithyIdentityAPI, .smithyHTTPAPI, .awsSDKCommon],
+            dependencies: [.awsSDKIdentityAPI, .crt, .smithy, .clientRuntime, .smithyIdentity, .smithyIdentityAPI, .smithyHTTPAPI, .awsSDKCommon, "InternalAWSSTS", "InternalAWSSSO", "InternalAWSSSOOIDC", ],
             path: "Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity"
         ),
+        .target(
+            name: "InternalAWSCommon",
+            dependencies: [.smithy, .smithyIdentity],
+            path: "Sources/Core/AWSSDKIdentity/Sources/InternalAWSCommon"
+        ),
```

Comment:
This is a very simple target with a dummy AWS credential identity resolver, not dependent on AWSSDKIdentity, that is used as the default identity resolver in the internal clients.

This target is not published, and is only linked to the internal clients.